x/leverage/client/tests/suite.go

```diff
@@ -712,6 +712,42 @@ func (s *IntegrationTestSuite) TestQueryBorrowAPY() {
 	runTestQueries(s, testCasesBorrowAPY)
 }
 
+func (s *IntegrationTestSuite) TestQueryMarketSize() {
+	testCasesMarketSize := []testQuery{
```

Comment:
This test can be improved by using `setup` commands with a `Lend` (and `cleanup` commands with a `Withdraw`). That way, the market size will be nonzero.

Note that the simulated `Umee` oracle price is 34.21 during CLI tests, so you should come out with an expected value of `sdk.MustNewDecFromStr("34.21")` if you lend `1000000uumee` in the setup.

Reply:
done at https://github.com/umee-network/umee/pull/376/commits/bcf42b0e98935e745ff0ae40024fd77e4230cee3

Reply:
add the clean up command in https://github.com/umee-network/umee/pull/376/commits/7a5c33f66a267aeb91567c7035a1e353c0af22d4